lib/placeholders/box/readMe.md

```diff
@@ -0,0 +1,30 @@
+## Navbar
+
+The box placeholder component.
+
+### Properties
+
+| Name    | Type     | Default   | Description                   |
+| :------ | :------- | :-------- | :---------------------------- |
+| `shape` | `String` | `default` | Prop to set the shape of box. |
+
+### Theme
+
+| Name        | Description                               |
+| :---------- | :---------------------------------------- |
+| `boxStyles` | Class used for the style the box element. |
+
+### Usage
+
+```
+
+const Demo = ({theme, shape}) => {
+    const classes = classnames(theme.styles, theme.boxStyles, theme[shape]);
+
+  const boxPlaceholderProps = {
+    className: classes,
+  };
+
+  return <div {...boxPlaceholderProps} />;
+}
```

Comment:
The Usage Section should be an example of how the component can be used.
```jsx
class Demo extends React.Component {
    state = {
       loading: false,
    }
    render() {
       const {loading} = this.state;
       return (
            <div>
                { loading ? <BoxPlaceholder /> : <div>User Content</div>
             </div>
        )
    }
}
```

Reply:
1309bc5f877031f4e7719566dc665b2940b86281